File: Day-32/day32.cpp
#include<iostream>
using namespace std;

int main(){
    int nums[6] = {1,2,3,4,5,6};


    for(int i = 0; i < 6; i++){
        int RG = -1;
        for(int j = i + 1; j < 6; j++){
            if(nums[j] > nums[i]){
                RG = nums[j];
                break;
        }
        }
        cout << RG << " ";
    }
    
    return 0;
}

File: Day-32/day32.py
nums = [1,2,3,5,6,7]

for i in range(len(nums)):
    RG = -1
    for j in range(len(nums[i + 1:])):
        if nums[j + i + 1] > nums[i]:
            RG = nums[j + i + 1 ]
            break
    print(RG)